### ISSUE TITLE
You can start Logstash with no worker and it will not complain.

### ISSUE THREAD

**Issue by ph** [elastic/logstash#5531]
When testing logstash, I was able to make logstash start with no worker.
Logstash doesn't complain it just sit waiting.

```
 bin/logstash -f ls_tools/generator.conf -w
```

```
~/e/logstash git:fix/pipeline-response-for-custom-metric ❯❯❯ bin/logstash -f ls_tools/generator.conf -w                                                                                                                                                                                                               ⏎ ✱ ◼
--- jar coordinate com.fasterxml.jackson.core:jackson-annotations already loaded with version 2.7.1 - omit version 2.7.0
--- jar coordinate com.fasterxml.jackson.core:jackson-databind already loaded with version 2.7.1 - omit version 2.7.1-1
Pipeline main started
```

I haven't check if we have validation of the setting for the worker.

**Comment by ph** [comment]
@jsvd could you take a look?

**Comment by jsvd** [comment]
closed by #5773
